.expo/packager-info.json

```diff
@@ -1,8 +1,8 @@
 {
   "expoServerPort": 19000,
   "packagerPort": 19001,
-  "packagerPid": 71664,
-  "expoServerNgrokUrl": "https://rd-khx.ijemmao.pongmates.exp.direct",
-  "packagerNgrokUrl": "https://packager.rd-khx.ijemmao.pongmates.exp.direct",
-  "ngrokPid": 71681
+  "packagerPid": 1110,
```

Comment:
I would go through this file and make sure that nothing has changed. I don't think you would want this file to change.

Reply:
ignore this comment